rtsf-at-checkout-rsp-controller-event-handler/res/configuration.toml

```diff
@@ -1,46 +1,54 @@
-# Copyright (C) 2019 - 2020 Intel Corp.
+# Copyright (C) 2019 - 2022 Intel Corp.
 # 
 # SPDX-License-Identifier: Apache-2.0
 
 [Writable]
 LogLevel = 'INFO'
 
 [Service]
-BootTimeout = '30s'
-ClientMonitor = '15s'
-CheckInterval = '10s'
+HealthCheckInterval = '10s'
 Host = 'localhost'
+MaxRequestSize = 0
+MaxResultCount = 0
 Port = 48096
-Protocol = 'http'
 ReadMaxLimit = 100
+RequestTimeout = '30s'
+ServerBindAddr = ""
 StartupMsg = 'RSP Event Handler App has started'
-Timeout = '30s'
 
 [Registry]
 Host = 'localhost'
 Port = 8500
 Type = 'consul'
 
 [Clients]
-  [Clients.CoreData]
+  [Clients.core-metadata]
   Protocol = "http"
   Host = "localhost"
-  Port = 48080
+  Port = 59881
 
 [Binding]
  Type="messagebus"
  SubscribeTopic="events"
 
-[MessageBus]
-Type = 'zero'
-    [MessageBus.SubscribeHost]
-        Host = 'localhost'
-        Port = 5563
-        Protocol = 'tcp'
-
-[Logging]
-EnableRemote = false
-File = ""
+[Trigger]
+Type="edgex-messagebus"
+  [Trigger.EdgexMessageBus]
+  Type = "mqtt"
+    [Trigger.EdgexMessageBus.SubscribeHost]
+    Host = "localhost"
+    Port = 1883
+    Protocol = "tcp"
+    SubscribeTopics="edgex/events/#"
+    [Trigger.EdgexMessageBus.Optional]
```

Comment:
Add `[Trigger.EdgexMessageBus.PublishHost]` so ResponseData is published to the MessageBus.
Topic will need placeholders so Reconciler subscribe topic matches
https://docs.edgexfoundry.org/2.3/microservices/application/Triggers/#publish-topic-placeholders
 `PublishTopic="edgex/events/{profilename}/{devicename}/{sourcename}"

Reply:
I have made the suggested change